runtime/commonMain/src/kotlinx/serialization/modules/SerializerAlreadyRegisteredException.kt

```diff
@@ -15,7 +15,7 @@ import kotlin.reflect.KClass
  * with [SerialModule.plus], consider using [SerialModule.overwriteWith] if you want overwriting behaviour.
  */
 @InternalSerializationApi // Will be hidden in the next release
-public class SerializerAlreadyRegisteredException private constructor(msg: String) : IllegalArgumentException(msg) {
+public class SerializerAlreadyRegisteredException internal constructor(msg: String) : IllegalArgumentException(msg) {
```

Comment:
You could move exception message to secondary constructors instead

Reply:
Nah, I'd prefer to keep the logic where it belongs